vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/DeviceActions.java

```diff
@@ -48,8 +48,17 @@ public DeviceActions(IWebDriverProvider webDriverProvider, WebDriverSessionInfo
     public void pushFile(String deviceFilePath, String resource)
     {
         byte[] fileBytes = ResourceUtils.loadResourceAsByteArray(getClass(), resource);
-        byte[] base64Bytes = Base64.getEncoder().encode(fileBytes);
+        pushBinaryData(deviceFilePath, fileBytes);
+    }
 
+    /**
+     * Pushes <b>binary data</b> onto a device at the <b>deviceFilePath</b> path
+     * @param deviceFilePath The destination file path on a device.
+     * @param data The binary data to push.
+     */
+    public void pushBinaryData(String deviceFilePath, byte[] data)
```

Comment:
```suggestion
    public void pushFile(String deviceFilePath, byte[] data)
```
also javadoc should be reworked